_middle/vacations.md

```diff
@@ -0,0 +1,78 @@
+weeks in each calendar year as far as possible, and EPFL HR really does not want
+you to carry vacation days over year to year.
+
+Various circumstances can additionally count as paid time off (see sources), of
+which the following are probably most relevant to PhD students:
+* your own marriage: 6 days
+* the marriage of a family member: 1 day
+* first aid or medical treatment of a member of your household: up to 3 days in each instance
+* if you are moving house: 1 day per calendar year
+* the death of a parent or member of your household: 5 days
+* the death of an other family member: up to 3 days
+
+([RS 172.220.113 art. 51](https://www.fedlex.admin.ch/eli/cc/2001/279/fr#art_51),
+[RS 172.220.113 art. 52](https://www.fedlex.admin.ch/eli/cc/2001/279/fr#art_52),
+LEX 4.1.4 [en](https://www.epfl.ch/about/overview/wp-content/uploads/2019/09/4.1.4_r_gestion_temps_travail_an.pdf)/[fr](https://www.epfl.ch/about/overview/wp-content/uploads/2019/09/4.1.4_r_gestion_temps_travail_fr.pdf))
+
+## Sickness and accidents
+
+If you fall sick or have an accidental injury, you are entitled to up to 3 days
+of paid leave without any special justification. You will be granted additional
+paid leave if you have a doctor’s note. There are limits on how long such leave
+can last and under what conditions; consult the laws for more details. Of
+course, paid leave ends if the contract ends, which is usually within a year for
+PhD students.
+
+([RS 172.220.113 arts. 36–36c](https://www.fedlex.admin.ch/eli/cc/2001/279/fr#art_36))
+
+## Parenthood
+
+If you have newly become a mother, you are entitled to 4 months of paid
+maternity leave, starting at most 1 month before the birth of the child. You can
+choose to convert the second half of your leave into reduced working hours with
+the agreement of your supervisor, e.g. 2 months at 50% instead of 1 month of
+full leave. Also, talk to your doctoral program head for details about stopping
+the ‘PhD clock’.
+
+If you have newly become a father, you are entitled to 20 days of paid paternity
+leave, of which 10 must be taken in the first six months after birth and the
+other 10 within the first year. Additionally, if both parents work in the EPF
+domain, the reduced working hours mentioned above can be shared between them.
+
+There are other provisions for adoption, illness of young children, etc. Consult
+the laws for more details. Sorry for the heteronormative language—that’s just
+how the laws are phrased (in French) and I didn’t want to introduce inaccuracies.
+
+([RS 172.220.113 arts. 47–47b](https://www.fedlex.admin.ch/eli/cc/2001/279/fr#art_47),
+[Maternity – EPFL HR](https://www.epfl.ch/campus/services/human-resources/en/maternity/),
+[Pregnancy, maternity, paternity, breastfeeding – EPFL Community & Support](https://www.epfl.ch/campus/community/family-and-work-life/pregnancy-maternity-paternity-breastfeeding/))
+
+## Remote work
+
+No academic discussion of vacations is complete without mentioning remote work…
```

Comment:
I'm a bit skeptical of this section because in practice it's much more flexible than this. I get that you wanted to stay close to the rules but IMHO the EPIC guide can afford to talk about how things are in practice, which is why it's more helpful than official EPFL pages.

I suggest rephrasing to some form of "talk to your advisor to know what they allow; if you cannot come to an agreement with them, know that..."